packages/actionmenu/src/react/index.js

```diff
@@ -32,7 +30,8 @@ ActionMenu.propTypes = {
 ActionMenu.defaultProps = {
   isKeyboarding: false,
   origin: vars.origins.topLeft,
-  shouldFocusOnMount: true
+  shouldFocusOnMount: true,
+  onClose: () => {}
```

Comment:
@jaketrent this was overriding the defaultProps I set in menu.js so I moved the onClose up to this indes.js file